### DISCUSSION TITLE
View tabular outputs in a presentable way

### DISCUSSION THREAD

**Discussion by Hrishikesh007788** [sinaptik-ai/pandas-ai#554] [Q&A]
I created an application that using pandasai. Currently I am taking the response generated and pasting it directly in my HTML frontend( {{ response }} ). Using tabulate might be an option. I would like to know if there are better options available.

**Comment by ronaldsholt** [comment]
Have you tried custom prompting? So, its _not_ totally reliable with AzureOpenAI from pandasai, but OpenAI class worked well for me using v1.0.11 (i assume v1.2 will too since there are so many problems with pandasai and Azure).... use cases like explain your findings are fun, but really depends... when it does work it looks / feels much better than just sending the results.

I too wanted a little more flex in the style of responses, and the ability to ground it for certain tasks. I hope more will be developed here bc its an important part of using this tool imo. LMK if you have questions!
